src/utils.js

```diff
@@ -84,6 +96,46 @@ export function useAppSetting(settingName) {
   return [settingValue, saveSetting];
 }
 
+export const validateAdvancedSettingsData = (object, setErrorFields) => {
```

Comment:
this function should not be located in global utils, please move it to `advanced-settings` folder